pycqed/measurement/multi_qubit_module.py

```diff
@@ -23,15 +23,18 @@
 import pycqed.measurement.detector_functions as det
```

Comment:
Logger needs name, I think we should refactor that into all files!

Reply:
I fixed it now by hand